drush/Commands/DeployCommands.php

```diff
@@ -565,7 +565,7 @@ public function waitForTaskToComplete($uuid) {
   public function newRelic($git_ref, $email, $application, $api_key) {
     $cmd = <<<EOT
 curl -X POST 'https://api.newrelic.com/v2/applications/$application/deployments.json' \
-     -H 'X-Api-Key:$api_key' -i \
```

Comment:
The key used is now a user key, and New Relic recommends sending keys with this header. New Relic no longer supports API keys